### DISCUSSION TITLE
How can I use with Laravel or Laravel Mix?

### DISCUSSION THREAD

**Discussion by solamichealolawale** [windicss/windicss#164] [General]
I've seen many other plugins for Vue and Svelte and all.
What about those of us coming from Tailwind with Laravel & Laravel Mix
This is how we have it configured.

```
mix.js('resources/js/app.js', 'public/js').vue()
    .postCss('resources/css/app.css', 'public/css', [
        require('postcss-import'),
        require('tailwindcss'),
        require('autoprefixer')
    ]).
    webpackConfig(require('./webpack.config'));

if (mix.inProduction()) {
    mix.version();
}
```

**Comment by i3acksp4ce** [comment]
I try this plugin but doesn't work, I don't know why.
https://github.com/windicss/windicss-webpack-plugin

Does anyone have any examples? Thanks.

**Comment by JenuelDev** [comment]
Try The windicss CLI, check this link it provides information on how to do it. this way you can preflight watch your blades instead. https://windicss.org/integrations/cli.html

**Reply by matthewjumpsoffbuildings** [reply]
I'm using laravel mix, but im using it to build `src/pug/*.pug` into `dist/html`.

doesnt windicss need to transform the output html files? for example to support variant grouping? so in my case i should run mix first to build the pug into html, then run windi CLI and point it at `dist/*.html`?

the issue with that is im also doing `.sass()` in my mix build step, and ideally i would want to have access to windi `@apply` etc in my sass files...

**Comment by JenuelDev** [comment]
I have found the way... it seems this is the best way to add, since laravel  latest are now using postcss. Just follow this in windicss website: https://windicss.org/integrations/postcss.html
